messenger.rst

```diff
@@ -1131,6 +1131,35 @@ during a request::
         :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`
         or :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\WebTestCase`.
 
+Amazon SQS
+~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The Amazon SQS transport has been added in Symfony 5.1
+    Install it by running:
+
+    .. code-block:: terminal
+
+        $ composer require symfony/amazon-sqs-messenger
+
+The ``SQS`` transport configuration looks like this:
+
+.. code-block:: bash
+
+    # .env
+    MESSENGER_TRANSPORT_DSN=sqs://guest:guest@sqs.eu-west-3.amazonaws.com/test?region=eu-west-3
+
+
+.. note::
+
+    By default, the transport will automatically create queue that are needed. That can be disabled.
```

Comment:
```suggestion
    By default, the transport will automatically create queues that are needed. This can be disabled.
```

```suggestion
    By default, the transport will automatically create queue that is needed. This can be disabled.
```

Reply:
First one is better.